doc/src/reference/py.rst

```diff
@@ -121,7 +121,10 @@ target
 
     Constructs the target.
 
-    :param str name: The name of the target to construct. This can either be 'cpu' or 'gpu'.
+    :param str name: The name of the target to construct. MIGraphx supports `cpu`, `gpu` and `ref` as targets. `ref` provides referecnce 
+    implementation to compare results for correctness against `cpu` or `gpu` targets.  By default `MIGraphX` that ships with `ROCm`, `cpu` target is disabled. 
+    So, If you are using `MIGraphX` from `ROCm`, then parameter `name` can only be either `gpu` or `ref`. 
```

Comment:
I dont understand what you are trying to link to. I dont think many of them are valid sphinx targets. Maybe you could make the role explicit to make it clearer.